paasta_tools/tron/client.py

```diff
@@ -109,3 +109,17 @@ def list_namespaces(self):
         """Gets the namespaces that are currently configured."""
         response = self._get('/api')
         return response.get('namespaces', [])
+
+    def get_job_content(self, job):
```

Comment:
Can you please add type annotations for args and the return value here and in other new/updated functions?

Reply:
Sure